Examples/Rendering/PBR/controller.html

```diff
@@ -35,3 +35,27 @@ <h3>PBR Properties</h3>
     </td>
   </tr>
 </table>
+<table>
+  <h3>Environment Properties</h3>
+  <tr>
+    <td>
+      <label>FOV: </label>
+      <input class='angle' type='range' min='30' max='120' value='30' step='1' oninput="this.nextElementSibling.value = this.value"/>
```

Comment:
I would default this to 75 to get a better looking background by default

Reply:
I'm having a strange issue where if I change the field of view before the initial rendering, zooming out immediately causes the entire screen to turn one color. I'm not sure why it's happening or how to fix it

Reply:
Weird.  I change the html to value=75 and it just seems to work, but maybe a graphics driver thing?